### PR TITLE
Python sdk

### PR DESCRIPTION
**Pull request by choir27**
Adds content, code samples, installation guide, and full example for Pieces Copilot Python SDK

Fixes https://github.com/pieces-app/documentation/issues/541
- https://github.com/pieces-app/documentation/issues/541

https://github.com/mason-at-pieces/pieces-copilot-python-sdk

### CODE REVIEW

File: docs/build/copilot/python/index.mdx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+After installing the SDK, you must initialize the SDK with your base URL. The base URL will depend on your setup.
+
+If you are using a local instance of Pieces OS:
+- On macOS/Windows, use http://localhost:1000
+- On Linux, use http://localhost:5323
+
+If you are using a remote instance of Pieces OS, use the URL you have set up for that.
+
+```python
+from pieces_copilot_sdk import PiecesClient
```
Comment (shivay-at-pieces) [inline docs/build/copilot/python/index.mdx:45]: Hey @choir27 this is wonderful! Just one small comment, at the moment, @Bishoy-at-pieces @mason-at-pieces @jimbobbennett are working on the Wrapper SDK implementation which will be added to the main Pieces Org. There is a complete rewrite of the wrapper SDK code being implemented right now which means that the implementation for the methods will change. But this is a good start. It also gives us good time to rethink how to actually structure the docs regarding the SDK. 

On a high level - we can think of initially introducing: what are the main category of APIs such as Assets APIs, Conversation APIs and have individual docs for each of these broad category alongside one main doc around the core SDK. 

Let me know your thoughts @mason-at-pieces

**Comment by Bishoy-at-pieces** [comment]
Absolutely great work @choir27 💪💪

File: docs/build/sdks/python/assets/index.mdx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+ **metadata** | **FragmentMetadata**| [optional]
+
+### Example
+
+```python
+from pieces_os_client.wrapper import PiecesClient
+from pieces_os_client import FragmentMetadata
+
+# Replace 'your_base_url' with the base URL of your Pieces OS server
+pieces_client = PiecesClient(config={'baseUrl': 'your_base_url'})
```
Comment (Bishoy-at-pieces) [inline docs/build/sdks/python/assets/index.mdx:30]: Just small update here @choir27 we change the config from baseUrl to host

```diff
-pieces_client = PiecesClient(config={'baseUrl':'your_base_url'})
+pieces_client = PiecesClient(config={'host':'your_base_url'})
```
It should be updated everywhere too

File: sidebars.ts [hunk 3]
```diff
@@ -569,20 +569,16 @@ const sidebars: SidebarsConfig = {
       id: 'build/examples/index',
       label: '📚 Examples',
     },
+    {
+      type: 'doc',
+      id: 'build/sdks/python/index',
+      label: '📚 Python',
```
Comment (shivay-at-pieces) [inline sidebars.ts:575]: Can we update the label with Python icon? Not sure if that is possible @mason-at-pieces @choir27

File: docs/build/sdks/python/index.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+## Features
+
+Feature | Description
+------------- | -------------
+Simplified Interaction | The Pieces Copilot SDK simplifies the interaction with the Pieces OS Client SDK by providing easy-to-use methods for various operations.
+Manage Conversations | The SDK provides various methods to manage conversations such as fetching a specific conversation, updating conversation name, and more.
+Copilot Chats | Communicate seamlessly with copilot chats functionality.
+Asset Management | Save and manage assets and formats efficiently.
+Local Server Interaction | Interact with a locally hosted server for various functionalities.
+Multi LLMs support | Use any Pieces supported LLMs to power apps.
```
Comment (shivay-at-pieces) [inline docs/build/sdks/python/index.mdx:26]: Use any of the Pieces supported offline and online LLMs

File: docs/build/sdks/python/quickstart/index.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+title: Pieces Wrapper Python SDK Quick Start
+description: Quickly set up the Pieces Wrapper Python SDK
+---
+
+# Python QuickStart
```
Comment (shivay-at-pieces) [inline docs/build/sdks/python/quickstart/index.mdx:6]: Quickstart

File: docs/build/sdks/python/quickstart/index.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+After installing the SDK, you must initialize the SDK with your base URL. The base URL will depend on your setup.
+
+If you are using a local instance of Pieces OS:
+- On macOS/Windows, use http://localhost:1000
+- On Linux, use http://localhost:5323
+
+If you are using a remote instance of Pieces OS, use the URL you have set up for that.
+
+## Getting started
```
Comment (shivay-at-pieces) [inline docs/build/sdks/python/quickstart/index.mdx:42]: We should mention about our Wrapper here. and how is it different than the base SDK.

File: docs/build/sdks/python/copilot.mdx [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+```python
+from pieces_os_client.wrapper import PiecesClient
+
+# Initialize the PiecesClient
+pieces_client = PiecesClient()
+
+# Get the chat ID
+chat_id = "your_chat_id"
+
+# Set the current chat
```
Comment (Bishoy-at-pieces) [inline docs/build/sdks/python/copilot.mdx:166]: You should give the BasicChat object here not the id

**Review by Bishoy-at-pieces — CHANGES_REQUESTED** [review]

File: docs/build/sdks/python/copilot.mdx (hunk 7 above)
Comment (mason-at-pieces) [reply docs/build/sdks/python/copilot.mdx:166]: Can you give an example of this?

Comment (Bishoy-at-pieces) [reply docs/build/sdks/python/copilot.mdx:166]: pieces_client.copilot.chat=BasicChat(id)

Comment (mason-at-pieces) [reply docs/build/sdks/python/copilot.mdx:167]: updated

File: docs/build/sdks/python/assets.mdx [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+
+### Properties
+
+| Property Name | Property Description |
+|--------------- |----------------------|
+| **id**        | The ID of the asset. |
+| **asset**    | The full [Asset](/build/reference/python/models/Asset) object. Should **only** be used for properties not available in the `BasicAsset` class. |
+| **raw_content** | The content of the asset. |
+| **is_image**   | Boolean indicating if the asset is an image. |
+| **classification** | The specific classification of the asset (e.g., `py`). |
```
Comment (Bishoy-at-pieces) [inline docs/build/sdks/python/assets.mdx:116]: Can we add here that this return the ClassificationEnum object and if you want the actual value you need to do `asset.classification.value`

**Review by Bishoy-at-pieces — APPROVED** [review]
LGTM

File: docs/build/sdks/python/assets.mdx (hunk 8 above)
Comment (mason-at-pieces) [reply docs/build/sdks/python/assets.mdx]: noted

**Pull request merged**